pandas/_libs/tslibs/parsing.pyx

```diff
@@ -47,6 +47,15 @@ cdef set _not_datelike_strings = {'a', 'A', 'm', 'M', 'p', 'P', 't', 'T'}
 
 # ----------------------------------------------------------------------
 
+_get_option = None
+
+def get_option(param):
+     global _get_option
```

Comment:
can you add a comment on what this is and why its here

Reply:
Done